Documentation/README_Solver.md

```diff
@@ -36,5 +38,19 @@ When *UpdateLinkedTransform* is true, the solver will calculate position & orien
 ## Expectations for extending or adding to the solver system ##
 To create a new solver script, you will want to extend from the abstract base class, *Solver*. This will ensure your new solver will tie into the state tracking and execution from both Solver and SolverHandler. The power with solvers comes from them being modular in nature, so as you extend Solver, keep this in mind. Its better to have many small solver scripts vs. one big one.
 
+## Hand constraint solvers ##
+
+The **HandConstraint** behavior provides a solver that constrains the tracked object to a region safe for hand constrained content (such as hand UI, menus, etc). Safe regions are considered areas that don't intersect with the hand. A derived class of **HandConstraint** called **HandConstraintPalmUp** is also included to demonstrate a common behavior of activating the solver tracked object when the palm is facing the user. For example use of this behavior please see the **HandBasedMenuExample** scene.
```

Comment:
Please xref HandConstraint & HandConstraintPalmUp.

FYI, @Cameron-Micka  I am working on a change that will update the SolverExample scenes and this documentation. The change are all for the above general solver information